source/server/http/admin.cc

```diff
@@ -663,7 +663,7 @@ AdminImpl::NullRouteConfigProvider::NullRouteConfigProvider()
 AdminImpl::AdminImpl(const std::string& access_log_path, const std::string& profile_path,
                      const std::string& address_out_path,
                      Network::Address::InstanceConstSharedPtr address, Server::Instance& server,
-                     Stats::Scope& listener_scope)
+                     Stats::ScopePtr&& listener_scope)
```

Comment:
Did you changed that?

Reply:
no, it is their change that in the big merge was lost. one of the reasons the previous PR did not compile.